### PR TITLE
zh: helm offline document

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/tidb-toolkit.md [hunk 1]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
+
+    ```shell
+    helm repo add pingcap https://charts.pingcap.org/
+    ```
 
-添加完成后，可以使用 `helm search` 搜索 PingCAP 提供的 chart：
+    添加完成后，可以使用 `helm search` 搜索 PingCAP 提供的 chart：
 
-如果 Helm 版本 < 2.16.0:
+    如果 Helm 版本 < 2.16.0:
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:266]:
```suggestion
    - 如果 Helm 版本 < 2.16.0:
```

File: zh/tidb-toolkit.md [hunk 2]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:268]:
```suggestion
        {{< copyable "shell-regular" >}}
```

File: zh/tidb-toolkit.md [hunk 3]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
 
-```shell
-helm search pingcap -l
-```
+    ```shell
+    helm search pingcap -l
+    ```
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:272]:
```suggestion
        ```shell
        helm search pingcap -l
        ```
```

File: zh/tidb-toolkit.md [hunk 4]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
 
-如果 Helm 版本 >= 2.16.0:
+    如果 Helm 版本 >= 2.16.0:
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:274]:
```suggestion
    - 如果 Helm 版本 >= 2.16.0:
```

File: zh/tidb-toolkit.md [hunk 5]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:276]:
```suggestion
        {{< copyable "shell-regular" >}}
```

File: zh/tidb-toolkit.md [hunk 6]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
 
-```shell
-helm search pingcap -l --devel
-```
+    ```shell
+    helm search pingcap -l --devel
+    ```
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:280]:
```suggestion
        ```shell
        helm search pingcap -l --devel
        ```
```

File: zh/tidb-toolkit.md [hunk 7]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
-pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
-...
-```
+    ```
+    NAME                    CHART VERSION   APP VERSION DESCRIPTION
+    pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
+    pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
+    pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
+    ...
+    ```
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:288]:
```suggestion
        ```
        NAME                    CHART VERSION   APP VERSION DESCRIPTION
        pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
        pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
        pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
        ...
        ```
```

File: zh/tidb-toolkit.md [hunk 8]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
-    helm install ${chart_name} --name=${release_name} --namespace=${namespace} --version=${chart_version} -f ${values_file}
+    helm del --purge ${release_name}
     ```
 
-* 执行升级（升级可以是修改 `chart-version` 升级到新版本的 chart，也可以是修改 `values.yaml` 文件更新应用配置）：
+    更多 helm 的相关文档，请参考 [Helm 官方文档](https://helm.sh/docs/)。
+
+4. 离线情况下配置 Helm repo
+
+    如果服务器上没有外网，就无法通过配置 Helm repo 来安装 tidb-operator 组件以及其他应用。这时，我们需要在有外网的机器上将集群安装用到的 chart 文件下载下来，拷贝到服务器上。
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:338]:
```suggestion
    如果服务器上没有外网，就无法通过配置 Helm repo 来安装 TiDB Operator 组件以及其他应用。这时，我们需要在有外网的机器上下载集群安装需用到的 chart 文件，再拷贝到服务器上。
```

File: zh/tidb-toolkit.md [hunk 9]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
+
+    我们可以通过下面命令将集群安装时候用到的 chart 下载下来：
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:340]:
```suggestion
    通过以下命令，下载集群安装时需要的 chart 文件：
```

File: zh/tidb-toolkit.md [hunk 10]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
-    helm upgrade ${release_name} ${chart_name} --version=${chart_version} -f ${values_file}
+    wget http://charts.pingcap.org/tidb-operator-v1.1.0.tgz
+    wget http://charts.pingcap.org/tidb-drainer-v1.1.0.tgz
+    wget http://charts.pingcap.org/tidb-lightning-v1.1.0.tgz
     ```
 
-最后，假如要删除 helm 部署的应用，可以执行：
-
-{{< copyable "shell-regular" >}}
+    将这些 chart 文件拷贝到服务器上并解压，然后就可以 `helm install` 使用这些 chart 来安装了，以 `tidb-operator` 为例：
```
Comment (ran-huang) [inline zh/tidb-toolkit.md:350]:
```suggestion
    将这些 chart 文件拷贝到服务器上并解压，可以通过 `helm install` 命令使用这些 chart 来安装相应组件，以 `tidb-operator` 为例：
```

File: zh/tidb-toolkit.md [hunk 11]
```diff
@@ -138,26 +138,94 @@ tidb-ctl schema in mysql
...
+    将 `tiller-rbac.yaml` 文件拷贝到服务器上并安装 `tiller`：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f tiller-rbac.yaml
+    helm init --service-account=tiller --skip-refresh
+    ```
+
+    Helm 服务端是一个名字叫 `tiller` 的服务，是作为一个 Pod 运行在 Kubernetes 集群里的。这个 Pod 使用的镜像是 `gcr.io/kubernetes-helm/tiller:v2.16.7`，如果无法访问 gcr.io，你可以尝试 mirror 仓库：
```
Comment (DanielZhangQD) [inline zh/tidb-toolkit.md:204]: v2.16.7 是可变的

File: zh/tidb-toolkit.md [hunk 12]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
     helm init --upgrade
     ```
 
-Kubernetes 应用在 Helm 中被打包为 chart。PingCAP 针对 Kubernetes 上的 TiDB 部署运维提供了多个 Helm chart：
+3. 配置 Helm repo
```
Comment (DanielZhangQD) [inline zh/tidb-toolkit.md:245]: 这里提一下没网络的话直接跳到 step 4?

File: zh/tidb-toolkit.md (hunk 11 above)
Comment (weekface) [reply zh/tidb-toolkit.md:204]: 这里是举例说明

File: zh/tidb-toolkit.md (hunk 12 above)
Comment (weekface) [reply zh/tidb-toolkit.md:245]: 都是一个 list 上下两项，无需频繁提及。

File: zh/tidb-toolkit.md [hunk 13]
```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
...
 
     ```shell
-    helm install ${chart_name} --name=${release_name} --namespace=${namespace} --version=${chart_version} -f ${values_file}
+    helm del --purge ${release_name}
     ```
 
-* 执行升级（升级可以是修改 `chart-version` 升级到新版本的 chart，也可以是修改 `values.yaml` 文件更新应用配置）：
+    更多 helm 的相关文档，请参考 [Helm 官方文档](https://helm.sh/docs/)。
+
+4. 离线情况下配置 Helm repo
```
Comment (DanielZhangQD) [inline zh/tidb-toolkit.md:336]:
```suggestion
4. 离线情况下使用 Helm chart
```

**Pull request merged**